frontend/src/assets/locales/en/dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+  "directorReview": {
```

Comment:
I think we also need to move all the dashboard widget texts here. It would be nice to include this as part of the database call improvement ticket.